tz.cpp

```diff
@@ -410,23 +410,25 @@ load_timezone_mappings_from_csv_file(const std::string& input_path)
     for (int i = 0; i < 4; ++i)
         getline(is, copyright);
     
```

Comment:
If you change it the way you think it should be, the changes will show up in this pull request.  I'd rather you do that than me guess and get it wrong.  Force-pushing is ok if you want to rewrite history.